### PR TITLE
replace gh-select-native with one-way-select

### PR DESCRIPTION
**Pull request by kevinansfield**
no issue
- removes `gh-select-native` component as ember-one-way-control's `one-way-select` component offers the same one-way-binding but also allows element attributes to be set

### CODE REVIEW

File: app/templates/components/modals/invite-new-user.hbs [hunk 1]
```diff
@@ -27,12 +27,13 @@
...
-                    content=roles
+                {{one-way-select
+                    id="new-user-role"
+                    options=roles
                     optionValuePath="id"
                     optionLabelPath="name"
-                    selection=role
-                    action="setRole"
+                    value=role
+                    action=(action "setRole")
```
Comment (acburdine) [inline app/templates/components/modals/invite-new-user.hbs]: needs to be `update=(action "setRole")`

File: app/templates/team/user.hbs [hunk 2]
```diff
@@ -115,12 +115,13 @@
...
-                                content=roles
+                            {{one-way-select
+                                id="new-user-role"
+                                options=roles
                                 optionValuePath="id"
                                 optionLabelPath="name"
-                                selection=model.role
-                                action="changeRole"
+                                value=model.role
+                                action=(action "changeRole")
```
Comment (acburdine) [inline app/templates/team/user.hbs]: same here

File: app/templates/components/modals/invite-new-user.hbs (hunk 1 above)
Comment (kevinansfield) [inline app/templates/components/modals/invite-new-user.hbs]: Interesting, I wonder why that didn't cause tests to fail

**Comment by kevinansfield** [comment]
Typos fixed, I think those areas weren't tested particularly well. I think if this was applied on top of #277 then the new tests would have at least picked up the invite modal typo.

**Pull request merged**